### PR TITLE
add some config啊

### LINKED ISSUE #8: issue8了
还好了

### PR DESCRIPTION
**Pull request by Jeffmen**
add some config了么我

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/app/build
+/api/build
+/.idea
+/.gradle
+/profile/build
+/library/plugin/build
```
Comment (jomatto) [inline .gitignore:6]: test

File: local.properties [hunk 2]
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
+#
+# Location of the SDK. This is only used by Gradle.
+# For customization when using a Version Control System, please read the
```
Comment (jomatto) [inline local.properties:5]: add test

File: local.properties [hunk 3]
```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
+# as it contains information specific to your local configuration.
```
Comment (jomatto) [inline local.properties:2]: good

**Review by jomatto — APPROVED** [review]

**Comment by Jeffmen** [comment]
test26

**Comment by Jeffmen** [comment]
稻妻

**Pull request closed without merging**